api_server.py

```
@@ -0,0 +1,253 @@
+        print(f"Prediction error: {e}")
+        return jsonify({'error': 'Internal server error'}), 500
+
+
+@app.route('/health', methods=['GET'])
+def health():
+    """Health check endpoint"""
+    return jsonify({'status': 'healthy', 'timestamp': datetime.now().isoformat()}), 200
+
+
+if __name__ == '__main__':
+    app.run(host='0.0.0.0', port=5000, debug=True)
```

Ruff (0.14.3)

Possible binding to all interfaces

(S104)